lib/robots/dor_repo/gis_assembly/extract_boundingbox.rb

```diff
@@ -51,12 +50,12 @@ def extent_shapefile(shpfn)
         # @return [Array#Float] ulx uly lrx lry
         def extent_geotiff(tiffn)
           LyberCore::Log.debug "extract-boundingbox: working on GeoTIFF: #{tiffn}"
-          IO.popen("#{Settings.gdal_path}gdalinfo '#{tiffn}'") do |f|
+          IO.popen("#{Settings.gdal_path}gdalinfo '#{tiffn}'") do |file|
             ulx = 0
             uly = 0
             lrx = 0
             lry = 0
```

Comment:
I left these alone for brevity.  "upper left x" "upper left y" "lower right x" "lower right y" could be used but I made a lot of changes already